programmers/20210511/예산.js

```diff
@@ -0,0 +1,9 @@
+function solution(d, budget) {
+    d.sort(function(a ,b) {
+        return a - b;
+    })
```

Comment:
arrow fuction을 이용하면 더 깔끔한 코드로 보입니다.
바로 return하는 경우에는 return이 생략가능합니다.
```suggestion
    d.sort((a, b) => a - b)
```